### PR TITLE
wip: periodics jobs for frozen/rotten/stale issues (bot)

### LINKED ISSUE #40: Lifecycle bot
Create a stale "bot" using a set of prowjobs in order to automatically tag the lifecycle phase of issues.

Currently, we have the stale bot (a GitHub action) that checks issues and pings authors if they haven't been updated in a while.
The way it's configured and the way it works is not clear neither transparent to our users.

In fact, we experienced some complaints in the past about it.

We should remove such bot and implement a better one to automate the lifecycle of issues (and maybe pull requests too) of the whole falcosecurity organization.

## Lifecycle

- frozen: issue or PR should not be autoclosed due to rottenness
- stale: issue or PR open with no activity (90days) and has become stale
- rotten: 30days have passed since the issue or the PR was marked as stale, it will be automatically closed soon
- close: after other 30days the rotten issue is automatically closed

Thus, 90+30+30 = an issue would be closed without _any_ activity after 150 days.

Using `/lifecycle frozen` an issue will be excluded from this process.

/cc @falcosecurity/test-infra-maintainers

### LINKED ISSUE #88: Implement lifecycle stale and rotten in Prow
**Motivation**

At the moment we use a GitHub bot to manage stale PRs and Issues, everyone is annoyed by that because it's not very configurable and it's difficult to manage from the issue/Pr itself.

We need something that is more flexible - using a ProwJob seem a good idea because we manage everything from a single point and because with ProwJobs you can essentially execute whatever you like while using a framework that has interaction points with issues and PRs.

**Feature**

To achieve that we need to:

- Enable [Crier](https://github.com/kubernetes/test-infra/blob/03be169fff10a03827ffe41dc65e314f239ac28b/prow/cmd/crier/README.md) and have it reporting back to GitHub - needed for ProwJobs to work correctly.
- Create a [ProwJob](https://github.com/kubernetes/test-infra/blob/ff6c355751962efd5d6b44f0f07567a9be1ea4f9/prow/jobs.md) to change the status if issues. A very good example is how the kubernetes community does this - see [here](https://github.com/kubernetes/test-infra/blob/ff6c355751962efd5d6b44f0f07567a9be1ea4f9/config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml#L180-L266)

We want to obtain something like this

**Alternatives**

Continue using the actual stale bot as a Github app.

**Additional context**

The actual deployments are in a Kubernetes cluster managed by the maintainers' team, whoever works on this will need to replicate the deployment on their own development machine and then send the PR - we don't do development directly on the prod infrastructure.

### PR DESCRIPTION
**Pull request by leodido**
Signed-off-by: Leonardo Di Donato <leodidonato@gmail.com>

This PR aims to introduce a bot which handles the lifecycle of falcosecurity projects. The ultimate goal is to substitute the stale github bot we have in place with something more smarter.

Fixes #40 
Fixes #88

### CODE REVIEW

File: prow/config/configs.yaml [hunk 1]
```diff
@@ -59,6 +59,109 @@ branch-protection:
...
 log_level: debug
 
+# todo > use placeholders in comment? https://github.com/kubernetes/test-infra/blob/888ff5ffe5991efad27997c7bd176e37a9a85b2f/robots/commenter/main.go#L44
+periodics:
+  - name: periodic-stale
+    interval: 1h
+    decorate: false
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/commenter:v20190528-0d7c4b53a
```
Comment (leodido) [inline prow/config/configs.yaml]: Update image?

File: prow/config/configs.yaml [hunk 2]
```diff
@@ -59,6 +59,109 @@ branch-protection:
...
+      - name: oauth
+        secret:
+          secretName: oauth-token
+
+  - name: periodic-close
+    interval: 1h
+    decorate: false
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/commenter:v20190528-0d7c4b53a
```
Comment (leodido) [inline prow/config/configs.yaml]: Update image?

File: prow/config/configs.yaml (hunk 1 above)
Comment (leodido) [inline prow/config/configs.yaml]:
```suggestion
      - image: gcr.io/k8s-prow/commenter:v20191202-824584dd0
```

File: prow/config/configs.yaml (hunk 2 above)
Comment (leodido) [inline prow/config/configs.yaml]:
```suggestion
      - image: gcr.io/k8s-prow/commenter:v20191202-824584dd0
```

**Comment by fntlnz** [comment]
Related to #88

**Comment by fntlnz** [comment]
I think this doesn't work without Crier @leodido

**Pull request closed without merging**